app/views/mailers/organisation_mailer/notify_rdv_changes.html.erb

```diff
@@ -1,23 +1,30 @@
 <p>Bonjour,</p>
 
-<p>Vous avez demandé à être notifié sur cette adresse email de tous les évènements liés aux rendez-vous (nouveaux rendez-vous, modifications et annulations de rendez-vous) de la catégorie <b>“<%= @motif_category.name %>”</b> de l’organisation <b>“<%= @organisation %>”.</b></p>
+<p>Vous avez demandé à être notifié sur cette adresse email de tous les évènements liés aux rendez-vous (nouveaux rendez-vous, modifications et annulations de rendez-vous) de la catégorie <b>“<%= @rdv.motif_category.name %>”</b> de l’organisation <b>“<%= @rdv.organisation %>”.</b></p>
 
 <p>Voici un nouvel évènement :</p>
 
 <p>
-  Évènement : <%= I18n.t("external_notifications.events_description.#{@event}") %><br />
-  Date : <%= I18n.l(@participation.rdv.starts_at, format: :human) %><br />
-  Usager : <%= @participation.user %><br />
-  <% if @participation.agent_prescripteur %><%= "Professionel : #{@participation.agent_prescripteur}" %><br /><% end %>
-  Motif : <%= @participation.rdv.motif_name %><br />
-  <% if @participation.rdv.lieu.present? %> <%= "Lieu : #{@participation.rdv.lieu.name}"%><br /> <% end %>
-  <% if @participation.rdv.lieu.present? %> <%= "Adresse : #{@participation.rdv.lieu.address}" %><br /><% end %>
-  Durée : <%= @participation.rdv.duration_in_min %> minutes<br />
+  <b>Évènement : <%= I18n.t("external_notifications.events_description.#{@event}") %></b><br />
+  Date du rdv : <%= I18n.l(@rdv.starts_at, format: :human) %><br />
+  Usager(s) concerné(s) :
 </p>
-<p class="btn-wrapper">
-  <%= link_to "Voir sur RDV-Insertion", organisation_user_follow_ups_url(@organisation.id, @participation.user.id), class: "btn btn-primary" %>
+<ul>
+  <% @rdv.participations.each do |participation| %>
```

Comment:
c'est sur les `@participations` que tu veux itérer je pense

Reply:
Ah merde yes bien vu